transip/service/dns.py

```diff
@@ -0,0 +1,50 @@
+"""
+DNS related objects
+"""
+
+from suds.sudsobject import Object as SudsObject
+
+
+class DnsEntry(SudsObject):
+    """
+    Representation of the DnsEntry object
```

Comment:
Since this is a simple class object, let's leave the comment out and change the pylint directive.